v1/fleet-local/control/booster-control@.service

```
@@ -0,0 +1,40 @@
+[Unit]
+Description=Booster Control @ %i
+Requires=docker.service
+After=docker.service
+
+[Service]
+User=core
+Restart=on-failure
+Environment="IMAGE=etcdctl get /images/booster"
+
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker pull $($IMAGE)"
+
+# mkdir so `etcdctl ls /booster/nodes` below doesn't fail
+ExecStartPre=-/usr/bin/sh -c "etcdctl mkdir /booster/nodes"
+
+# Set our internal hostname minus .ec2.internal so
+# ip-123.456.789.123.ec2.internal becomes ip-123.456.789.123 which is what
+# inet:gethostname() reports
+#
+# This is only done for control tier nodes because they're expected to stick
+# around and epmd will create a full mesh without explicity registering worker
+# nodes
+ExecStartPre=/usr/bin/sh -c "etcdctl set /booster/nodes/$(echo %H | cut -d'.' --complement -s -f2-) true"
+
+ExecStartPre=-/usr/bin/docker kill booster
+ExecStartPre=-/usr/bin/docker rm booster
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+  --name booster \
+  --net=host \
+  -e BOOSTER_ROLES=control_tier \
+  -e BOOSTER_NODES=$(etcdctl ls /booster/nodes | cut -d'/' --complement -s -f-3 | tr '\n' ',') \
```

More specifically: [this will fail silently](https://github.com/adobe-platform/booster-erl/blob/ab51c434d27399720b152c8226d6703a6c55251a/src/bstr_app.erl#L56) and the `nodes()` api won't include failed nodes